tests/integration/test_cli.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+from unittest import mock
+
+from fastapi_security import cli
+
+
+def test_usage_output_without_params():
+    result = subprocess.run(["fastapi-security"], capture_output=True)
+    assert result.returncode == 2
+    assert result.stdout.decode().splitlines() == []
+    assert result.stderr.decode().splitlines() == [
+        "usage: fastapi-security [-h] {gendigest} ...",
+        "fastapi-security: error: the following arguments are required: subcommand",
+    ]
+
+
+def test_usage_with_help_param():
+    result = subprocess.run(["fastapi-security", "-h"], capture_output=True)
+    assert result.returncode == 0
+    assert result.stdout.decode().splitlines() == [
+        "usage: fastapi-security [-h] {gendigest} ...",
+        "",
+        "fastapi_security command-line interface",
+        "",
+        "positional arguments:",
+        "  {gendigest}  Specify a sub-command",
+        "",
+        "optional arguments:",
+        "  -h, --help   show this help message and exit",
+    ]
+    assert result.stderr.decode().splitlines() == []
+
+
+def test_gendigest_without_params():
+    result = subprocess.run(["fastapi-security", "gendigest"], capture_output=True)
+    assert result.returncode == 2
+    assert result.stdout.decode().splitlines() == []
+    assert result.stderr.decode().splitlines() == [
+        "usage: fastapi-security gendigest [-h] --salt SALT",
+        "fastapi-security gendigest: error: the following arguments are required: --salt",
+    ]
```

Comment:
`fastapi-security` command is not found when I run `poetry run pytest`. Perhaps call `cli.main` directly like you've done in `test_gendigest_smoke_test`? E.g:

```suggestion
def test_gendigest_without_params(capsys):
    with pytest.raises(SystemExit, match="2"):
        cli.main(["gendigest"])

    captured = capsys.readouterr()
    assert captured.out == ""
    assert captured.err.splitlines() == [
        "usage: fastapi-security gendigest [-h] --salt SALT",
        "fastapi-security gendigest: error: the following arguments are required: --salt",
    ]
```